starter_code/css/styles.css

```diff
@@ -0,0 +1,87 @@
+header{
+    background-color: black;
+    color: white;
+    overflow: hidden;
+    height: 100px;
+    border-bottom: 20px solid yellow;
+}
+.logo
+{
+    color: yellow;
+
+}
+nav ul li {
+    display: inline;
+    color: white;
+    
+
+}
+.HStartup{
+    color: yellow;
+}
+header nav a{
+     color:white;
+     display: block;
```

Comment:
`display: block` property is overwrite `display: inline` in `nav ul li` selector.
So delete `display` property here to make the `nav` links display inline.